openstef/model/serializer.py

```diff
@@ -536,3 +531,10 @@ def _get_feature_names(
                     pid=pid,
                 )
         return modelspecs.feature_names
+
+
+def _get_model_uri(artifact_uri: str) -> str:
+    """Set model uri based on latest run.
+    Note that this function is primarily useful
+    so it can be mocked during unit tests"""
```

Comment:
Nice to clean up how we get the path to the model! If we only use this `get_model_uri` function for mocking, then I think we could also mock os.path.join and lose the extra function for unit tests potentially. Will be a bit cleaner

Reply:
Good suggestion. I've kept it as-is for now, I like it better as a separate function, but I guess that's just taste:)